templates/confirmation-email-sent.html

```diff
@@ -1,37 +1,47 @@
-{% extends 'layouts/_base.html' %}
-{% from 'components/panel/_macro.njk' import onsPanel %}
+{% extends "layouts/_base.html" %}
+
+{% from "components/panel/_macro.njk" import onsPanel %}
+
 {% set page_title = _("Confirmation email sent") %}
 {% set hide_sign_out_button = content.hide_sign_out_button %}
 {% set sign_out_url = content.sign_out_url %}
 
 {% block main %}
-   {% call onsPanel({
-     "spacious": true,
-     "type": "success",
-     "classes": "ons-u-mb-s",
-     "iconType": "check",
-     "iconSize": "xl"
-     })
-   %}
-  <h1 data-qa="confirmation-text">{{ _("A confirmation email has been sent to {email}").format(email = content.email)  }}</h1>
- {% endcall %}
-
-  {% call onsPanel({
-      "type": "bare",
-      "classes": "ons-u-mb-s",
-      "iconType": "lock"
-      })
-  %}
-      <p>{{ _("The email will be sent from <em>census.2021@notifications.service.gov.uk</em>") }}</p>
-  {% endcall %}
-
-  {% if content.show_send_another_email_guidance %}
-    <p><strong>{{ _("Didn't receive an email?")  }}</strong></p>
-    <p class="ons-u-mb-l">{{ _("It can take a few minutes for the email to arrive. If it doesn't arrive, check your junk mail, or you can <a href='{send_confirmation_email_url}' id='send-another-email'>send another confirmation email</a>.").format(send_confirmation_email_url = content.send_confirmation_email_url)}}</p>
-  {% endif %}
-
-  {% if content.show_feedback_call_to_action %}
-    {% include 'partials/feedback-call-to-action.html' %}
-  {% endif %}
-
+    {# djlint:off #}
+    {% call
+        onsPanel({
+            "spacious": true,
+            "type": "success",
+            "classes": "ons-u-mb-s",
+            "iconType": "check",
+            "iconSize": "xl"
+        })
+    %}
+        <h1 data-qa="confirmation-text">
```

Comment:
Would we want to turn on djlint for before the `<h1>` and `<p>` (a bit further on) blocks? Although that might mean another set of `{# djlint:off #}` / `{# djlint:on #}` in this file 🤔

Reply:
I tried, the problem here is `<h1> `and `<p>` are part of the DS component call (`{% call onsPanel`), they are nested inside the call, and the formatter is flattening them when not excluded so might have to keep it as it was.